### PR TITLE
Update list-a-project.md

### PR DESCRIPTION
**Pull request by Rinsington-DC**
Brand spelling

### CODE REVIEW

File: docs/list-a-project.md [hunk 1]
```diff
@@ -48,7 +47,7 @@ tags:
   - the
   - project
 upforgrabs:
-  name: up for grabs
+  name: Up-For-Grabs
```
Comment (shiftkey) [inline docs/list-a-project.md:50]: This change is not correct, as it's part of the project configuration and the casing here is important

**Comment by shiftkey** [comment]
@Rinsington-DC can we address this all in one PR? I added another example to #1657, and I want to pause and ensure we settle on one naming of it.

**Comment by Rinsington-DC** [comment]
Hi, sure. I wasn't aware I could do them all in one PR!

File: docs/list-a-project.md (hunk 1 above)
Comment (Rinsington-DC) [reply docs/list-a-project.md:50]: Good call!

**Comment by shiftkey** [comment]
> Hi, sure. I wasn't aware I could do them all in one PR!

Simply ensure you're viewing the branch associated with the previous PR, which I think is this link: https://github.com/Rinsington-DC/up-for-grabs.net/tree/patch-2

Then find the file you want to change and open it in the web editor.

**Comment by shiftkey** [comment]
Closing this out in favour of #1657

**Pull request closed without merging**